### PR TITLE
xcenv 1.0.4

### PR DESCRIPTION
**Pull request by maschall**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Resubmitting after some gained attention (https://iosdevweekly.com/)

### CODE REVIEW

File: Formula/xcenv.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  def install
+    prefix.install ["bin", "libexec"]
+  end
+
+  def caveats; <<-EOS.undent
+    Xcenv stores data under ~/.xcenv by default. If you absolutely need to
+    store everything under Homebrew's prefix, include this in your profile:
+      export XCENV_ROOT=#{var}/xcenv
+    To enable shims and autocompletion, run this and follow the instructions:
+      xcenv init
```
Comment (dunn) [inline Formula/xcenv.rb]: This second part isn't Homebrew-specific information, is it?  That's all we allow in the caveats.

Comment (maschall) [inline Formula/xcenv.rb]: The tool is very much based off rbenv, so I followed the rbenv formula (https://github.com/Homebrew/homebrew-core/blob/master/Formula/rbenv.rb).

I am happy to remove it, if it's not allowed.

It is useful information for the user on how to further setup the tool, but I can see what you mean.

Comment (dunn) [inline Formula/xcenv.rb]: Yeah, let's remove it.  We should probably drop the same line from rbenv.

The rationale is that (1) upstream might change how it works and we'll be giving people old information and (2) the more text in the caveats, the less likely they'll be read.

Comment (maschall) [inline Formula/xcenv.rb]: Makes sense, removed

**Comment by tdsmith** [comment]
previously https://github.com/Homebrew/homebrew-core/pull/3096

**Comment by tdsmith** [comment]
Thanks for your pull request! :tada: Homebrew thrives because of contributions like this one.

**Pull request closed without merging**